src/main/java/httpServer/RequestResponseHandler.java

```diff
@@ -0,0 +1,27 @@
+package httpServer;
+
+import java.io.BufferedReader;
+import java.io.PrintWriter;
+
+public class RequestResponseHandler {
+    private final BufferedReader input;
+    private final PrintWriter output;
+
+    public RequestResponseHandler(BufferedReader input, PrintWriter output) {
+        this.input = input;
+        this.output = output;
+    }
+
+    public void run() {
+        receiveRequest();
+        sendResponse("HTTP/1.1 200 OK");
```

Comment:
How do you know what status to send? Should it be hardcoded here going forward?

Reply:
It was hardcoded to get the tests to pass for this story only. It will not be hardcoded going forward